example/src/main.ts

```diff
@@ -12,12 +12,18 @@ const client = new SignalClient(window.origin)
 // WebRTC Configuration
 const RTC_CONFIGURATION = {
     iceServers: [
+        {
+            urls: "stun:stun.relay.metered.ca:80",
```

Comment:
Didn't realize the config included a `stun` server, let's use the `google` servers for `stun`